packages/venia-concept/src/components/MiniCart/cartOptions.js

```diff
@@ -4,9 +4,7 @@ import { Form } from 'informed';
 import { Price } from '@magento/peregrine';
 
 import { mergeClasses } from 'src/classify';
-import LoadingIndicator, {
-    loadingIndicator
```

Comment:
There is still a reference to `loadingIndicator` used on line 84 as a suspense fallback. Here's the thing -- we changed the `LoadingIndicator` component to export two things: a static indicator that renders as a full page (`fullPageLoadingIndicator`) or a `LoadingIndicator` component that accepts styles. I'm not sure the cart will need the full page loading indicator so you will probably want to create an inline static one here using `LoadingIndicator and pass it to the component on line 84.

For example:

```jsx
const loadingIndicator = (<LoadingIndicator><span>{'Fetching Options...'}</span></LoadingIndicator>);

...

<Suspense fallback={loadingIndicator}...
```

Here's an example of how we do it in `editItem`: https://github.com/magento-research/pwa-studio/blob/develop/packages/venia-concept/src/components/MiniCart/editItem.js#L11-L13

Reply:
yeah i see. Updated PR